classes/mender-boot.bbclass

```diff
@@ -0,0 +1,5 @@
+EXTRA_IMAGEDEPENDS += "u-boot u-boot-fw-utils"
+PREFERRED_VERSION_u-boot = "v2015.10%"
+PREFERRED_VERSION_u-boot-fw-utils = "v2015.10%"
+
+IMAGE_INSTALL_append = " u-boot-fw-utils"
```

Comment:
Will it be possible to just inherit one class and use `mender-boot` only? The reason I am asking is that it is no longer obvious which one you should include in your configuration to have Mender up and running.